### PR TITLE
chore(spanner): release 1.13.0

### PR DESCRIPTION
**Pull request by larkee**
This pull request was generated using releasetool.

### CODE REVIEW

File: spanner/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,26 @@
...
+11-11-2019 15:59 PST
+
+
+### Implementation Changes
+Fix TransactionPingingPool to stop thowing ''NoneType' object is not callable' error. ([#9609](https://github.com/googleapis/google-cloud-python/pull/9609))
+Return sessions from pool in LIFO order. ([#9454](https://github.com/googleapis/google-cloud-python/pull/9454))
+
+### New Features
+
+### Dependencies
```
Comment (tseaver) [inline spanner/CHANGELOG.md]: Please remove the empty sections.

File: spanner/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,26 @@
...
+
+### Documentation
+- Add Python 2 sunset banner to documentation. ([#9036](https://github.com/googleapis/google-cloud-python/pull/9036))
+- Update description of the timeout_secs parameter. ([#9381](https://github.com/googleapis/google-cloud-python/pull/9381))
```
Comment (tseaver) [inline spanner/CHANGELOG.md]:
```suggestion
- Update description of the `timeout_secs` parameter. ([#9381](https://github.com/googleapis/google-cloud-python/pull/9381))
```

File: spanner/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,26 @@
...
+
+### Internal / Testing Changes
+- Harden 'test_transaction_batch_update*' systests against partial success + abort. ([#9579](https://github.com/googleapis/google-cloud-python/pull/9579))
```
Comment (tseaver) [inline spanner/CHANGELOG.md]:
```suggestion
- Harden `test_transaction_batch_update*` systests against partial success + abort. ([#9579](https://github.com/googleapis/google-cloud-python/pull/9579))
```

**Comment by larkee** [comment]
@tseaver Changes made

**Pull request merged**